imports/plugins/core/orders/client/templates/workflow/shippingSummary.js

```diff
@@ -111,6 +111,16 @@ Template.coreOrderShippingSummary.helpers({
       }
     });
 
+    const canceled = _.every(shipment.items, (shipmentItem) => {
```

Comment:
I'd avoid import lodash and using _ if there is an ES6 equivalent.  (array.every prototype?).